File: 1023 - Administración Financiera/problem_1023.java
import java.text.DecimalFormat;
import java.util.Scanner;


public class problem_1023 {
    public static void main(String[] args) {
        Scanner e = new Scanner(System.in);
         
        DecimalFormat df = new DecimalFormat("#.00");
        double[] mes = new double[12];
        double total=0;
        for(int i=0; i<12; i++)
        {
            mes[i] = e.nextDouble();
        }
        for(double suma:mes)
        {
            total+=suma;
        }
        
        System.out.println("$"+df.format(total/12));
    }
}

File: 1023 - Administración Financiera/problem_1023.cs
using System;

namespace problem_1023
{
    class Program
    {
        static void Main(string[] args)
        {
            decimal[] mes = new decimal[12];
            decimal total=0;
            for(int i=0; i<12; i++)
            {
                mes[i] = decimal.Parse(Console.ReadLine());
            }
            foreach(decimal dec in mes)
            {
                total+=dec;
            }
            Console.WriteLine("$"+(total/12));
        }
    }
}
